go.mod

```diff
@@ -4,6 +4,7 @@ go 1.17
 
 require (
 	github.com/coreos/go-systemd v0.0.0-20191104093116-d3cd4ed1dbcf // indirect
+	github.com/coreos/go-systemd/v22 v22.3.2
```

Comment:
Is it weird that we have a different go-systemd and go-systemd/v22?  How does that work?

And it also doesn't look like you checked in the vendored version of this or the go-cmp that's required below - I think after running 'go mod vendor' you also need to `git add -a vendor/` and check that in.